Client/Frontend/Browser/TabManager.swift

```diff
@@ -249,7 +249,9 @@ extension TabManager {
         }
 
         let selectedIndex: Int = coder.decodeIntegerForKey("selectedIndex")
-        self.selectTab(self.tabs[selectedIndex])
+        if (selectedIndex >= 0) {
```

Comment:
@b-andris I'm going to grab this PR and fix it up so that we can merge it. Thanks for your contribution!